spec/models/lti_configuration_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe LtiConfiguration, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You could add a shoulda matcher test like so: https://github.com/education/classroom/blob/1f81278aca77b523d8b821ce206d6a35e523217e/spec/models/invite_status_spec.rb#L11

Suggestion:
```suggestion
  it { should belong_to(:organization) } 
```